docs/src/app/components/pages/components/IconMenu/ExampleNested.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+
+import MoreVertIcon from 'material-ui/lib/svg-icons/navigation/more-vert';
+import IconMenu from 'material-ui/lib/menus/icon-menu';
+import MenuItem from 'material-ui/lib/menus/menu-item';
+import IconButton from 'material-ui/lib/icon-button';
+import ArrowDropRight from 'material-ui/lib/svg-icons/navigation-arrow-drop-right';
+
+const IconMenuExampleNested = () => (
+  <div>
+    <IconMenu
+      iconButtonElement={<IconButton><MoreVertIcon/></IconButton>}
+      anchorOrigin={{horizontal: 'left', vertical: 'top'}}
+      targetOrigin={{horizontal: 'left', vertical: 'top'}}
+    >
+      <MenuItem
+        primaryText="Copy & Paste"
+        rightIcon={<ArrowDropRight />}
+        menuItems={[
+          <MenuItem primaryText="Cut" />,
+          <MenuItem primaryText="Copy" />,
+          <MenuItem primaryText="Paste" />,
+        ]}
+      />
+
+      <MenuItem
+        primaryText="Case Tools"
+        rightIcon={<ArrowDropRight />}
+        menuItems={[
+          <MenuItem primaryText="UPPERCASE" />,
+          <MenuItem primaryText="lowercase" />,
+          <MenuItem primaryText="CamelCase" />,
+          <MenuItem primaryText="Propercase" />,
+        ]}
+      />
+
```

Comment:
What do you think about add a `MenuItem` without a `menuItems` property in this example?